### PR TITLE
Feature cron

### LINKED ISSUE #63: Schedule tasks to run at a set date and time

### LINKED ISSUE #64: Verify completion of scheduled jobs

### PR DESCRIPTION
**Pull request by jehov4**
Closes #63 and Closes #64

### CODE REVIEW

File: 02_operation_of_running_systems/060_schedule_tasks/061_schedule_tasks.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Schedule Tasks
+This chapter is based of chapter 14.3 of Linux- Rheinwerk.
```
Comment (philippabele) [inline 02_operation_of_running_systems/060_schedule_tasks/061_schedule_tasks.md:2]: ... is based on ...

File: 02_operation_of_running_systems/060_schedule_tasks/061_schedule_tasks.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- Hour
+- Day of Month
+- Month
+- Day of Week
+
+An asterisk `*` means, that the task will be run every unit of time (minute, hour, day, ...). A task scheduled with `17 5  *  *  *` will run every month, every day, on hour *5* and minute *17*.
+The time frame is followed by the user which the task belongs to. The entry is ended with the actual command to be run. 
+
+### The Crontab Tool
+The crontab files are not to be edited directly, instead the tool `crontab` is used. The following flags are useful when working with the tool:
```
Comment (philippabele) [inline 02_operation_of_running_systems/060_schedule_tasks/061_schedule_tasks.md:39]: ... file are not intended to be edited directly ...

File: 02_operation_of_running_systems/060_schedule_tasks/062_check_for_completion.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Check if a Task was Executed
+The cron daemon logs the execution of tasks to syslog. As a consequence the execution of the tasks can be seen in the system journal.
+
+An example taken from [here](https://bencane.com/2011/11/02/did-my-cronjob-run/) below:
```
Comment (philippabele) [inline 02_operation_of_running_systems/060_schedule_tasks/062_check_for_completion.md:4]: please cite the important commands

**Review by philippabele — CHANGES_REQUESTED** [review]

File: 02_operation_of_running_systems/060_schedule_tasks/061_schedule_tasks.md (hunk 1 above)
Comment (jehov4) [reply 02_operation_of_running_systems/060_schedule_tasks/061_schedule_tasks.md]: fixed

File: 02_operation_of_running_systems/060_schedule_tasks/062_check_for_completion.md (hunk 3 above)
Comment (jehov4) [reply 02_operation_of_running_systems/060_schedule_tasks/062_check_for_completion.md:10]: added journalctl command call

**Review by philippabele — APPROVED** [review]

**Pull request merged**